quizup/src/Routes/BrowseMusic/BrowseMusic.css

```diff
@@ -0,0 +1,38 @@
+.BrowseMusic-section {
+  display: flex;
```

Comment:
suggestion: centering items using flex is commonly used. hence, can be made into a utility class